content/en/docs/hardware/graphic-card.md

```diff
@@ -10,16 +10,21 @@ weight: 7
 ## Reference Links
 * [LED Strain {{< param slug >}} Tag](https://ledstrain.org/t/{{< param slug >}}?sort=top)
 
--DISABLE DITHERING IN NVIDIA CARDS:
+## Dithering
+### Nvidia
 
-There is a parameter called "DIGITAL VIBRANCE" which is mainly the most important parameter to take in consideration when trying to disable dithering. As higher the number higher the use of dithering. It is used mainly for gamers, but even them consider it is not good to have it turned so high for much long.
+* [Disabling color banding in Nvidia](https://youtu.be/DOGVm1tjsT4)
 
-Depending on the Windows 10 version it is easilly removed or not. It could be good to watch the following tutorial to edit the registry to get rid of dithering completelly. Don't miss it at the end of the video an explanation regarding problems you can find in build numbers newer than 1607: https://youtu.be/DOGVm1tjsT4
+Nvidia cards use a technology called `Digital Vibrance` which may be related to dithering in Nvidia cards.  
+In Windows 10 newer then build 1607, it may be possible to disable by disabling color banding.
 
--DISABLE DITHERING IN AMD CARDS:
+### Amd Cards
 
-The dithering parameter related in AMD cards is the one called saturation. Reducing it it should reduce the eyestrain.
+The dithering parameter related in AMD cards is the one called saturation.
 
--DISABLE DITHERING IN INTEL CARDS:
+### Intel Cards
 
-There is no parameter to be disabled on Control panel to get rid of dithering on Intel cards, so we have to use 3rd party applications. In windows we can use "ditherig" and in Linux igfxtweak can be used, as for sure another ways to achieve it. Those 3rd party apps could work on Nvidia or AMD cars, as well
+Intel's included Control Panel does not currently allow configuration of dithering.  
+Some 3rd party applications are available that may allow dithering to be configured.  
+* Windows: `ditherig`
+* Linux: `igfxtweak`
```

Comment:
igfxtweak isn't a Linux utility.

Reply:
If I was going to merge any part of this pull request I'd atleast mention the `ditherig` utility for Intel graphics with a disclaimer saying "results may very". Too many uncertainties with the AMD and Nvidia sections.

Reply:
Yeah, I was trying to rephrase it into something that would fit better.